spec/lib/gakubuchi/task_spec.rb

```diff
@@ -38,17 +38,17 @@
         end
       end
 
-      context 'precompiled pathname is not nil' do
-        let(:precompiled_pathname) { double(:pathname) }
+      context 'digest path is not nil' do
+        let(:digest_path) { double(:path) }
```

Comment:
[IMO]
`double(:pathname)` seems better because it gives better information on the original object.